libutils/logging.c

```diff
@@ -173,7 +173,7 @@ static void LogToConsole(const char *msg, LogLevel level, bool color)
         fprintf(stdout, "%s ", formatted_timestamp);
     }
 
-    fprintf(stdout, "%s: %s\n", LogLevelToString(level), msg);
+    fprintf(stdout, "%8s: %s\n", LogLevelToString(level), msg);
```

Comment:
It could be `%7s`, if saving one character is important, I have no problem with that. :-)